packages/demo-form/src/pages/FormPage/form-page.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Factor } from "@slashid/slashid";
+import { Form, ConfigurationProvider } from "@slashid/react";
+import styles from "../../styles/Page.module.css";
```

Comment:
Importing CSS modules not collocated with the component is an indicator of a possibly missing component. Do you think it would make sense to create a `PageLayout` component that would look like this:

```
export function FormPage({containerClassName}) {
  return (
    <main className={styles.page}>
      <section className={styles.section}>
        <div className={containerClassName}>
          {children}
        </div>
      </section>
    </main>
  );
}
```

So then the FormPage would be something like:

```
<PageLayout>
  <ConfigurationProvider factors={factors}>
    <Form />
  </ConfigurationProvider>
</PageLayout>
```